frame/staking/src/lib.rs

```diff
@@ -427,6 +427,21 @@ pub enum StakerStatus<AccountId> {
 	Nominator(Vec<AccountId>),
 }
 
+/// Policy for receiving rewards
+#[derive(PartialEq, Eq, Copy, Clone, Encode, Decode, RuntimeDebug)]
+pub enum RewardPolicy<AccountId> {
+	/// Pay into single RewardDestination
+	One(RewardDestination<AccountId>),
+	/// Pay (Perbill * Total) to 1st Dest, ((1-Perbill) * Total) to 2nd Dest
+	Split(RewardDestination<AccountId>, Perbill, RewardDestination<AccountId>),
```

Comment:
I don't like the idea of splitting rewards to 2 destinations based on the impact it will have to the cost of making a payout. Basically we will double the worst case scenario cost for a payout, which will effectively reduce number of payouts by half.

I would say any such splitting of staked/unstaked funds should happen within a single account unless we can be more clever about weights here.